components/BookCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Heading, Image, Text } from "@chakra-ui/react";
+import { useBookQuery } from "@lib/hooks/book";
+import { Book } from "@prisma/client";
+import { useRouter } from "next/router";
+
+export default function BookCard({ name, imageUrl, isbn }: Book) {
+  const { isLoading, data: populatedBook } = useBookQuery(isbn);
+  const router = useRouter();
+
+  const authors = (): string => {
+    if (isLoading) {
+      return "Loading Authors..."
+    }
+    if (populatedBook?.googleBook?.authors) {
+      return populatedBook.googleBook.authors.join(', ')
+    }
+    return "No Authors"
```

Comment:
Do we want to explicitly state there are no authors? Might be cleaner just to have empty field or a "—" or something?